### ISSUE TITLE
Remove videos from limit count

### ISSUE THREAD

**Issue by jakubstraka** [michael-lynch/instagram-lite#11]
When I set limit to 10 and 5 of them are videos, I see only 5 images. Would be great to load 10 images no matter how many videos are in. Or show videos as images.

**Comment by michael-lynch** [comment]
Yeah, I agree, that is how the plugin should work. I'm just not sure what the best approach for that would be. I could retrieve their entire feed and then slice the array to fit the count, but that seems like such a waste of resources. I'll have to explore this when I have some time.

**Comment by blaindy** [comment]
+1
